code/__DEFINES/items_clothing.dm

```diff
@@ -112,6 +112,7 @@
 // Inventory slot strings.
 // since numbers cannot be used as associative list keys.
 //icon_back, icon_l_hand, etc would be much better names for these...
+#define slot_belt_str		"slot_belt"
```

Comment:
Is this bit actually still used, considering we figured out the icon_override thing?

Reply:
Yeah I saw that, but I couldn't remember if we used it or not. If we didn't I'll remove it.

Reply:
A quick look through shows that nothing uses it, so yeah.